network-api/networkapi/wagtailpages/templates/wagtailpages/tags/cta.html

```diff
@@ -3,7 +3,3 @@
 {% if cta_type == 'Petition' %}
 	{% include './cta/petition.html' %}
 {% endif %}
-
-{% if cta_type == 'Signup' %}
```

Comment:
Removing this code from this file but leaving the check for Petition here makes it hard to trace what's supposed to render at the end of the day. Instead of removing this chunk of code, can you replace the content of `signup.html` with [approximately these lines](https://github.com/mozilla/foundation.mozilla.org/blob/10d2a12881a237cee7baaaf9cbb319715e7b8363/network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html#L63-L73) and then point to this CTA template? It feels like that'll be a whole lot cleaner to trace in the future.